doc/man1/openssl-x509.pod.in

```diff
@@ -80,7 +80,7 @@ B<openssl> B<x509>
 
 =head1 DESCRIPTION
 
-This command is a multi purpose certificate utility. It can
+This command can
```

Comment:
I'd probably replace utility with tool and keep the sentence.

Reply:
done.